src/main/java/in/org/projecteka/hiu/valuseset/ValueSetController.java

```diff
@@ -0,0 +1,18 @@
+package in.org.projecteka.hiu.valuseset;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Mono;
+
+@RestController
+@AllArgsConstructor
+public class ValueSetController {
+    private ValueSetResource valueSetResource;
+
+    @GetMapping(value = "ValueSet", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
you probably have to configure/map the HIU UI api call as well

Reply:
On that.